configure

```diff
@@ -16189,6 +16193,25 @@ fi
 
 CXXFLAGS="$CXXFLAGS $CFLAGS"
 
+if test x$enable_shared = xyes; then
+    ENABLE_SHARED_TRUE=
+    ENABLE_SHARED_FALSE="#"
+else
+    ENABLE_SHARED_TRUE="#"
+    ENABLE_SHARED_FALSE=
+fi
+if test x$enable_static = xyes; then
+    ENABLE_STATIC_TRUE=
+    ENABLE_STATIC_FALSE="#"
+else
+    ENABLE_STATIC_TRUE="#"
+    ENABLE_STATIC_FALSE=
+fi
+
+
+
+
+
```

Comment:
Can you remove all these blank lines?

Reply:
This file is generated by autotools, so I don't think there's much that can be done about that.

Reply:
Ah, ok. Thanks for clarifying.